bin/activate_haddock

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+export HADDOCK3_NUM_CORES=4
 export HADDOCKPATH=`pwd`
 export PYTHONPATH=$PYTHONPATH:$HADDOCKPATH
 export PATH=$PATH:$HADDOCKPATH/bin
+export HADDOCK3_CNS_EXE="${HADDOCKPATH}/bin/cns/cns"
```

Comment:
is here not `bin/cns`?

Reply:
`bin/cns/cns` is created during the steps provided in the installation, see `README`. In a future PR I will change this as well. but for now we can keep that behaviour not to loose the scope. Here, I simply renamed the executable for MAC to a more general `cns` name. See `README`.

Reply:
fair point, it makes no sense to be `/bin/cns/cns`

Reply:
You are right. it can be just on `bin`.